lib/plugins.js

```diff
@@ -119,53 +125,56 @@ function plugins (settings, deviceId) {
       return _.map(r => r[0] + r[1].provisioned, _.zip(sum, tx.bills))
     }
 
-    const provisioned = _.reduce(sumTxs, [0, 0], redeemableTxs)
-    const zipped = _.zip(_.map('count', cassettes), provisioned)
+    const provisioned = _.reduce(sumTxs, Array(_.size(orderedCassettes)).fill(0), redeemableTxs)
```

Comment:
what do you think about doing it this way just to follow the pattern you've been using? 
```suggestion
    const provisioned = _.reduce(sumTxs, _.times(_.constant(0), _.size(orderedCassettes)), redeemableTxs)
```